dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/snowflake/AbstractSnowflakeConnectorTest.java

```diff
@@ -105,4 +105,13 @@ public void openConnection_failsForMixedPrivateKeyAndPassword() throws IOExcepti
             .contains(
                 "Private key authentication method can't be used together with user password"));
   }
+
+  @Test
+  public void checkJnaInClasspath_success() {
+    try {
+      Class.forName("com.sun.jna.Library");
+    } catch (ClassNotFoundException e) {
+      Assert.fail("net.java.dev.jna was not found in the classpath.");
```

Comment:
Please add the motivation for this lib. Something like `required for Snowflake mfa`

Reply:
done, I added comment and note in a message.